### ISSUE TITLE
About qpic v-coco eval result.

### ISSUE THREAD

**Issue by alexw994** [JacobYuan7/OCN-HOI-Benchmark#3]
Hello, I noticed that in the article, the eval result for qpic in V-COCO is not 58.8 as mentioned in the original paper but 61.8. Is there a specific reason for this difference, and could it be related to the training datasets of V-COCO (based on COCO 2014) and DETR (trained on COCO 2017)? Can you please explain the reason for this? Thanks!
![image](https://github.com/JacobYuan7/OCN-HOI-Benchmark/assets/37922390/0a64eee9-620d-4a18-a56f-2c3517d653f4)

**Comment by JacobYuan7** [comment]
@alexw994  Because in QPIC, the authors indicate that "A few previous works inappropriately use COCO train2017 set for pre-training, whose images are contained in the V-COCO test set.". So, QPIC pre-trains the detection models by himself without those images. Thus, they obtain lower performance. However, all other papers uses the COCO detection pre-training. Thus, I re-train the model to obtain performance under the setting that can be compared to other methods. This is a more fair comparison, which might be ignored in other papers.

**Comment by alexw994** [comment]
@JacobYuan7 
If my understanding is correct, the qpic score of 61.8 in the paper results from training on the V-COCO dataset using pretraining parameters from DETR (COCO train2017), is that accurate? Similarly, the OCN score of 64.2 is achieved through a comparable process.

**Comment by JacobYuan7** [comment]
@alexw994 
Yes, your understanding is precise. QPIC is my baseline in a sense. I did this for fair comparison in this table.
